src/FamilyHubs.Referral.Core/Services/IDistributedCacheService.cs

```diff
@@ -0,0 +1,22 @@
+﻿using FamilyHubs.Referral.Core.Models;
+using FamilyHubs.ServiceDirectory.Shared.Dto;
+
+namespace FamilyHubs.Referral.Core.Services;
+
+public interface IDistributedCacheService
+{
+    //Service
+    ServiceDto? RetrieveService();
+    void StoreService(ServiceDto serviceDto);
+
+    //Navigation - last page name
+    string RetrieveLastPageName();
+    void StoreCurrentPageName(string? currPage);
```

Comment:
Would you mind removing methods that are not used yet?
(It's hard to review code outside of the scope of the current story.)